### DISCUSSION TITLE
Osu Lazer sensitivity bug (spinner abuse 7k rpm)

### DISCUSSION THREAD

**Discussion by ShinyWise** [ppy/osu#19981] [Q&A] [answered]
https://youtu.be/-2cZMBQrQCk (not a rickroll im promise)
just want to share it to you 
cant explain it but I know you smart enough to understand this

**Comment by frenzibyte** [comment, marked as answer]
Spinner abuse/cheesing is known, see https://github.com/ppy/osu/issues/8224.
